lib/ast-link-to-transform.js

```diff
@@ -29,13 +29,19 @@ class AngleBracketLinkToPolyfill {
     let b = this.syntax.builders;
     let { moduleName } = this;
 
-    // in order to debug in https://https://astexplorer.net/#/gist/0590eb883edfcd163b183514df4cc717
+    // in order to debug in https://astexplorer.net/#/gist/0590eb883edfcd163b183514df4cc717/6284c6222fe671a1be16432477cdf83c840dfcf0
     // **** copy from here ****
     function transformAttributeValue(attributeValue) {
       switch (attributeValue.type) {
         case 'TextNode':
           return b.string(attributeValue.chars);
         case 'MustacheStatement':
+          if (attributeValue.params.length) {
+            // Deals with helper usage
+            return b.path(
+              b.sexpr(attributeValue.path.original, attributeValue.params, null, attributeValue.loc)
```

Comment:
You are not passing `hash` here. But yes, this is the culprit, my initial implementation was too naive.

Also I realized the original generic transform for angle brackets also has to deal with transforming attributes obviously, and has this figured out properly already: https://github.com/rwjblue/ember-angle-bracket-invocation-polyfill/blob/master/lib/ast-transform.js#L40-L53.

So I think it would be best to extract this into another helper function, and reuse it across all three transforms (including the one for `<Input>`, which suffers from the same problem I think).